package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.2",
```

Comment:
- [ ] Kindly write a descriptive README for your project. You can follow this Microverse [template](https://github.com/microverseinc/readme-template).